CondFormats/GEMObjects/interface/GEMeMap.h

```diff
@@ -19,7 +19,7 @@ class GEMeMap {
   void convertDummy(GEMROMapping& romap);
 
   struct GEMChamberMap {
-    std::vector<unsigned int> fedId;
+    std::vector<int> fedId;
```

Comment:
I hope this is a transparent change for streaming. I recall that the conditions formats are not really modifiable in an easy way (no schema evolution)

Reply:
i will just revert back